### PR TITLE
MODEXPW-474 - "Request has expired" or  "Expired token" errors

### PR DESCRIPTION
**Pull request by obozhko-folio**
[MODEXPW-474](https://folio-org.atlassian.net/browse/MODEXPW-474) - "Request has expired" or  "Expired token" errors

## Purpose
Fix "Expired token" errors.

## Approach
Populate filenames.

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [ ] Check logging
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by siarhei-charniak — APPROVED** [review]

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     },
     {
       "id": "erm",
-      "version": "7.0"
+      "version": "6.1"
```
Comment (alekGbuz) [inline descriptors/ModuleDescriptor-template.json:39]: Hi @obozhko-folio version of interface was decreased, could you check is this correct.

Comment (obozhko-folio) [reply descriptors/ModuleDescriptor-template.json:39]: this is temporary for testing purpose

File: src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java [hunk 2]
```diff
@@ -150,12 +153,27 @@ private void processJobUpdate(JobExecution jobExecution, boolean after) {
...
       jobExecutionUpdate.setProgress(progress);
     }
+    populateFileNames(jobExecution, jobExecutionUpdate, jobParameters);
     kafka.send(KafkaService.Topic.JOB_UPDATE, jobExecutionUpdate.getId().toString(), jobExecutionUpdate);
     if (after) {
       log.info("-----------------------------JOB---ENDS-----------------------------");
     }
   }
 
+  private void populateFileNames(JobExecution jobExecution, Job jobExecutionUpdate, JobParameters jobParameters) {
```
Comment (alekGbuz) [inline src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java:163]: @obozhko-folio , that logic should be updated if you are not going to check type in scope of the mod-data-export-spring

Comment (obozhko-folio) [reply src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java:163]: I returned back checking export type in mod-data-export-spring

Comment (NikitaSedyx) [reply src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java:163]: you shouldn't check type in mod-data-export-spring

**Review by alekGbuz — APPROVED** [review]

**Pull request merged**